challenge.md

```diff
@@ -6,40 +6,27 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+git clone (repo SSH url)
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+git log
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+git commit --ammend
 
 4. It looks like there was a change on the original repo that we forked from. How could we grab those changes and incorporate them into our local repo?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+To get the changes we need to use the command git pull upstream.
 
 5. Suppose that we wanted to look at the third-to-last commit on `master`; what command(s) would we write to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+git checkout --HEAD~3
```

Comment:
No `--`, but otherwise right.

(Actually, to be even more precise, we could do `git checkout master~3`, since `git checkout HEAD~3` will depend on where `HEAD` currently is.)